lib/iris/tests/__init__.py

```diff
@@ -115,7 +115,9 @@
     NC_TIME_AXIS_AVAILABLE = False
 
 try:
-    requests.get('https://github.com/SciTools/iris')
+    # Added a timeout to stop the call to requests.get hanging on the HPC
+    # which has restricted/no internet access.
```

Comment:
"on the HPC" makes the comment a bit too specific to the Met Office. Could you make it more general and say "a platform with no internet access"

So something like:
```
# Added a timeout to stop the call to requests.get hanging when running
# on a platform which has restricted/ no internet access.
```

Reply:
Sorry to barge in, but just a couple of queries ..
 * I think I understood that this is known to be Python3-specific problem ? 
   Are we sure of that / why that is / it is definitely appropriate to add an explicit timeout here (and fix does not belong here)
 * are we clear that 1 second is actually long enough ?  It sounds possibly a bit short to me.

Reply:
No problem i'll update the message.
When testing this I found the cut off point to be about 0.6 seconds so chose 1 second as a timeout, however any advice on a more appropriate fix / timeout value would be appreciated.

Reply:
Our Travis runs don't always run with good resourcing, we frequently get test run timeouts where something that "normally" takes 5-6minutes stretches to 10-15.
So I'd say it's handy to be generous, maybe 5-10 seconds ?
I guess the downside of an over-early test failure would be, for us, if it silently misses a test which it should have run, and something unintended might slip through.

Reply:
Changed timeout to 10 seconds.